### PR TITLE
Revisits sanitize, ports sanitize from TauCeti

### PR DESCRIPTION
**Pull request by quardbreak**

### CODE REVIEW

File: code/_global_vars/lists/names.dm [hunk 1]
```diff
@@ -13,3 +13,16 @@ var/global/list/verbs =              file2list("config/names/verbs.txt")
 var/global/list/adjectives =         file2list("config/names/adjectives.txt")
 //loaded on startup because of "
 //would include in rsc if ' was used
+
+var/global/list/forbidden_names = list(
```
Comment (MistakeNot4892) [inline code/_global_vars/lists/names.dm]: This seems a bit pointless on our codebase.

**Review by MistakeNot4892 — COMMENTED** [review]
Generally nothing in this leaps out at me as erroneous, but I don't have the understanding to approve properly.

@Lohikar could you have a look?

**Comment by MistakeNot4892** [comment]
I'll approve this if you update the conflicts, Lohi hasn't commented so I'm going to assume it's fine.

**Comment by Lohikar** [comment]
I'll take a look at this today, kind of forgot about it.

File: code/_global_vars/lists/names.dm (hunk 1 above)
Comment (Lohikar) [reply code/_global_vars/lists/names.dm]: Was this comment addressed?

File: code/_global_vars/lists/names.dm [hunk 2]
```diff
@@ -13,3 +13,16 @@ var/global/list/verbs =              file2list("config/names/verbs.txt")
...
+	"space",
+	"floor",
+	"wall",
+	"r-wall",
```
Comment (Lohikar) [inline code/_global_vars/lists/names.dm]: what year is it

**Comment by MistakeNot4892** [comment]
Stale PR, please reopen when updated.

**Comment by afterthought2** [comment]
Reopened by author's request.

**Review by afterthought2 — COMMENTED** [review]
Seems fine, actually. We should just go for it at this point.

**Review by afterthought2 — APPROVED** [review]

**Pull request merged**